app/views/return-requirements/remove.njk

```diff
@@ -24,7 +24,7 @@
       <p class="govuk-body govuk-!-font-weight-bold govuk-!-font-size-27"> {{ returnRequirement }}</p>
 
       <p class="govuk-!-margin-bottom-7"></p>
-      {{ govukButton({ text: "Confirm cancel" }) }}
+      {{ govukButton({ text: "Remove" }) }}
```

Comment:
Sorry, if you are making this change could you also apply the prevent double click attribute at the same time?

```suggestion
      {{ govukButton({ text: "Remove", preventDoubleClick: true }) }}
```